tests/core/pyspec/eth2spec/test/electra/block_processing/test_process_consolidation_request.py

```diff
@@ -580,6 +580,8 @@ def test_incorrect_same_source_target(spec, state):
     # Set source and target to be the same
     target_index = source_index
     source_address = b"\x22" * 20
+    # Switch target to compounding credential
+    # to make it a valid consolidation request
     set_compounding_withdrawal_credential_with_balance(
```

Comment:
Hmm not quite what I was looking for. Something more like:

```suggestion
    # Make source/target a compounding validator (0x02) to so the this isn't a valid switch to compounding request.
    # To be a valid switch to compounding request, the source validator must be an eth1 validator (0x01).
    set_compounding_withdrawal_credential_with_balance(
```

https://github.com/ethereum/consensus-specs/blob/83639a6bad3ee8fb046ac7284560038614d06b56/specs/electra/beacon-chain.md?plain=1#L1609-L1611